zapcore/sampler_test.go

```diff
@@ -37,6 +37,7 @@ import (
 
 func fakeSampler(lvl LevelEnabler, tick time.Duration, first, thereafter int) (Core, *observer.ObservedLogs) {
 	core, logs := observer.New(lvl)
+	// Keep using deprecated constructor for cc.
```

Comment:
what is `cc` and why can't this be updated?

Reply:
code coverage